Libraries/pbxspec/Headers/pbxspec/PBX/ProductType.h

```diff
@@ -65,8 +65,29 @@ class ProductType : public Specification {
         bool parse(plist::Dictionary const *dict);
     };
 
+public:
```

Comment:
A specification is a `.xcspec` file that defines various things, like available build settings, compiler flags, or known file types. A product type is a specification that defines a kind of thing built by a target, like an "Application" or a "Static Library". In the Xcode UI they're under the "+" icon in the list of targets:

<img width="747" alt="screen shot 2016-10-13 at 11 22 03" src="https://cloud.githubusercontent.com/assets/163390/19361384/544221ce-9137-11e6-94e3-85d7f864ed0e.png">